devops-case-study.md

```diff
@@ -0,0 +1,88 @@
+---
+title: DevOps case study
+keywords: development platforms, automation, repetitive tasks, processing
+tags: DevOps, software development methodology, principles, optimize
+sidebar: development-platforms
+permalink: development-platforms/devops-case-study.md
+folder: development-platforms
+---
+
+---
+
+## Introduction
+
+DevOps is a software development methodology that combines the principles of development (Dev) and operations (Ops) to create a streamlined, collaborative approach to software delivery. 
+The goal of DevOps is to automate and optimize the entire software delivery process, from planning and development to testing, deployment, and maintenance.
+
+## Brief History
+
+Provide a brief history of the product, including major milestones or events.
+- 1957 using Fortran, first computer programmer
+- 1969 ARPANET launches the first network engineering job and network operation center
+- 2003: Google had the first set of "site reliability engineers"
+- DevOps started to coalesce between 2007 and 2008.
+- John Allspaw(Senior Vice-President of Technical Operations) & Paul Hammond(Director of Engineering) presented in 2009 a presentation on a case why communication and collaboration between development and operations are vital, which is now called DevOps. 
+- 2009 Patrick Debois organizes the first DevOps day.
+- 2012 Spluink and Solarwinds was some of the first DevOps tools
+- 2013 Puppet labs releases the first state of a DevOps Survey. 
+- 2017 AI-powered server monitoring and centralized logging under one login.
+
+## Features
```

Comment:
Features, Stengths and Weaknesses doesn't apply so well to a philosophy like DevOps